specifyweb/frontend/js_src/lib/components/AppResources/tree.ts

```diff
@@ -67,13 +67,25 @@ function getGlobalAllResources(resources: AppResources): {
   const globalDirectories = resources.directories.filter(
     (directory) => directory.scope === 'global'
   );
-  if (globalDirectories.length === 0)
-    globalDirectories.push({
-      ...addMissingFields('SpAppResourceDir', {
-        userType: 'Common',
-      }),
-      scope: 'global',
-    });
+
+  const directoriesNeeded = ['Global Prefs', 'Prefs', 'Common'];
+  if (globalDirectories.length < directoriesNeeded.length) {
```

Comment:
this condition should be removed - this assumes that there are only 3 possible user types at a global level - not an assumption I am willing to make given that these directories are created by sp6 - who knows what it does